src/test/kotlin/uk/gov/justice/digital/hmpps/hmppstier/integration/TierCalculationTest.kt

```diff
@@ -27,22 +27,6 @@ class TierCalculationTest : MockedEndpointsTestBase() {
     }
   }
 
```

Comment:
Duplicate of 
Scenario: Sentence with out of date assessment has change level 2